README.md

```diff
@@ -44,6 +44,27 @@ To report an artifact deployment to Faros specify `CD` in the `event` parameter
     run-ended-at: 1594948069000 # millis since epoch, ISO-8601 string or 'Now'
 ```
 
+### Report a code deployment (CD Event) To Faros with commit
+
+To report the deployment of a commit to Faros specify `CD` in the `event` parameter and include the `CD` required fields.
+
+```yaml
+- name: Report deployment to Faros
+  id: send-cd-event
+  uses: faros-ai/faros-cicd-github-action@v3.0.10
+  with:
+    api-key: ${{ secrets.FAROS_API_KEY }}
+    event: CD
+    commit: GitHub://my-org/my-repo/sha
```

Comment:
`commit:` might be confusing, as it implied just `sha`. perhaps we should allow passing all the args in parts?

Reply:
otherwise, consider renaming to `commit-uri` or something